### PR TITLE
AppStore Release Edits

### PR DESCRIPTION
**Pull request by rs929**
## Overview

- Edits to release Scooped to the App Store

### CODE REVIEW

File: Scoop/AppDelegate.swift [hunk 1]
```diff
@@ -40,3 +57,37 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+        // If you are receiving a notification message while your app is in the background,
+        // this callback will not be fired till the user taps on the notification launching the application.
+        // TODO: Handle data of notification
+        // With swizzling disabled you must let Messaging know about the message, for Analytics
+        // Messaging.messaging().appDidReceiveMessage(userInfo)
+        // Print message ID.
+        // Print full message.
+        print(userInfo)
+        return UIBackgroundFetchResult.newData
+    }
```
Comment (vinnie4k) [inline Scoop/AppDelegate.swift:92]: Add line below

File: Scoop/AppDelegate.swift [hunk 2]
```diff
@@ -40,3 +57,37 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
 }
+
+extension AppDelegate: UNUserNotificationCenterDelegate {
```
Comment (vinnie4k) [inline Scoop/AppDelegate.swift:61]: Add line below

**Review by vinnie4k — APPROVED** [review]
LGTM

**Review by tiffany-pann — APPROVED** [review]
LGTM 🏎

**Pull request merged**